### DISCUSSION TITLE
Docker, Update SAE

### DISCUSSION THREAD

**Discussion by NewUser1Z** [camueller/SmartApplianceEnabler#224] [Installation und Betrieb]
Hallo,
ich bin Software Laie und konnte dank der Dokumentation (Copy-Paiste Anleitung) den Docker installieren und den SAE zum Laufen bringen. (Saubere Arbeit Axel!)

Aktuell läuft jetzt 1.6.19.
Ich würde jetzt gerne auf 2.0.2 Updaten. Gibt es hier auch eine Anleitung? (Bitte steinigt mich nicht, wenn ich diese übersehen habe.)

**Comment by camueller** [comment]
Normale Doku für Manuelle Installation auf dem 2.0-Branch: https://github.com/camueller/SmartApplianceEnabler/blob/2.0/doc/ManualInstallation_DE.md

**Comment by NewUser1Z** [comment]
Hallo Axel,
vielen Dank für den Link.
Ich habe mir das mehrfach durchgelesen, leider reichen meine Kenntnisse hier nicht und ich bin bei dieser Art von Update gescheitert.

Dein Update (1.6.20) gestern hat mich aber weitergebracht.
Ich habe folgendes probiert und es hat auch funktioniert.
pi@raspberrypi:~ $ **sudo bash**
root@raspberrypi:/home/pi# **docker pull avanux/smartapplianceenabler-arm32**
Jetzt läuft die 1.6.20 und alle Geräte sind auch noch da.

Um auf die Version 2.0.2 zu kommen habe ich folgendes probiert:
root@raspberrypi:/home/pi# **docker pull avanux/smartapplianceenabler-arm32:2.0.2**
Das hat leider nichts gebracht, es läuft immernoch die 1.6.20.

Gibt es die Möglichkeit dem Docker zu sagen, dass er den Container 2.0.2 nehmen soll?

**Comment by camueller** [comment]
Auch wenn ich mich wiederhole: Ich verstehe nicht, warum jemand ohne SAE-Kenntnisse (und offensichtlich auch ohne Docker-Kenntnisse) gleich einen virtualisierten SAE betreiben will. Bring doch erstmal ein einfaches Setup mit der offizielen 1.6er-Version zum Laufen. Wenn das läuft, kannst Du virtualisieren.
Du kannst es auch anders machen, aber bitte erwarte dann keinen Support von mir.
